.github/workflows/ci.yaml

```
@@ -1,952 +1,720 @@
-          echo "| cargo-tests | ${{ needs.cargo-tests.result }} |" >> $GITHUB_STEP_SUMMARY
-          echo "| wasm-pack-tests | ${{ needs.wasm-pack-tests.result }} |" >> $GITHUB_STEP_SUMMARY
-          echo "| docs | ${{ needs.docs.result }} |" >> $GITHUB_STEP_SUMMARY
-          echo "| webview-tests | ${{ needs.webview-tests.result }} |" >> $GITHUB_STEP_SUMMARY
-          # echo "| miri-tests | ${{ needs.miri-tests.result }} |" >> $GITHUB_STEP_SUMMARY  # TEMPORARILY DISABLED
-          echo "| prof-concurrency | ${{ needs.prof-concurrency.result }} |" >> $GITHUB_STEP_SUMMARY
-          echo "| proto-sync | ${{ needs.proto-sync.result }} |" >> $GITHUB_STEP_SUMMARY
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo "::error::One or more CI jobs failed!"
-          exit 1
+    - name: Report failure
+      run: |
```

actionlint (1.7.12)

[error] property "miri-tests" is not defined in object type {cargo-tests: {outputs: {}; result: string}; docs: {outputs: {}; result: string}; prek: {outputs: {}; result: string}; prof-concurrency: {outputs: {}; result: string}; proto-sync: {outputs: {}; result: string}; release-metadata: {outputs: {}; result: string}; wasm-pack-tests: {outputs: {}; result: string}; webview-tests: {outputs: {}; result: string}}

(expression)